resources/server/utils/generateProfileName.ts

```diff
@@ -16,7 +16,7 @@ export async function generateProfileName(
   const phone_number = clean(player.getPhoneNumber());
 
   if (firstname && lastname) {
-    return `${firstname}${delimiter}${lastname}`;
+    return `${firstname}${delimiter}${lastname}${identifier}`;
```

Comment:
Why do we want the identifier here?

Reply:
Even if this is not related to this PR (this is now deleted), just to answer : 
I'm currently using the multichar system as a multi simcard system, each "npwd character" is a new simcard in my framework, so identifier is the simcardId in my system. Adding identifier in this profile will fix a problem I have, if you have a character with multiple simcard, they will have the same profile name by default and this will lead to errors.